libafl/src/executors/command.rs

```diff
@@ -489,6 +490,7 @@ impl CommandExecutorBuilder {
     /// Adds a range of arguments to the program's commandline.
     ///
     /// You may want to use [`CommandExecutor::parse_afl_cmdline`] if you're going to pass `@@`
+    /// represents the input file generated by the fuzzer.
```

Comment:
The line breaks are not in the compiled doc, so it'll look like this: 
`if you're going to pass `@@` represents the input file generated by the fuzzer.`
I don't want to nag too much, but maybe we should write something more clear, like
```
if you're going to pass `@@` to represent the input file generated by the fuzzer (similar to the `afl-fuzz` command line).
```

Reply:
> The line breaks are not in the compiled doc

Yes, that's what I want.

> but maybe we should write something more clear

Done, please let me know if any improvements, it's really important for newcomers!

Reply:
Me neither😄, both are acceptable to me. Keep the current since this pr has been merged.